blog/zk-snark-pallet.md

```diff
@@ -0,0 +1,263 @@
+# Zk-SNARKs with Substrate (Part 3)
```

Comment:
Dodajemy też zdanie do tytułu, jak w części nr 2? Coś w stylu: Zk-SNARKs in Substrate (Part 2). Implementing pallets

Reply:
I would go with: "Pallet implementation"